pkg/ctl/cmdutils/scale.go

```diff
@@ -0,0 +1,91 @@
+package cmdutils
+
+import (
+	"fmt"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+)
+
+// NewScaleNodeGroupLoader will load config or use flags for 'eksctl scale nodegroup'
+func NewScaleNodeGroupLoader(cmd *Cmd, ng *api.NodeGroup, ngFilter *NodeGroupFilter) ClusterConfigLoader {
+	l := newCommonClusterConfigLoader(cmd)
+
+	l.flagsIncompatibleWithConfigFile.Insert(
+		"nodes",
+		"nodes-min",
+		"nodes-max",
+	)
+
+	validateNodeGroup := func(ng *api.NodeGroup) error {
```

Comment:
This could be reused, lets extract it to a function called something like `validateNumberOfNodes()` or `validateCapacityValues()` or something like that.